packages/dcos-ui-update-client/src/utils.ts

```diff
@@ -0,0 +1,12 @@
+import { RequestResponse } from "@dcos/http-service";
+
+export function getErrorMessage(reqResp: RequestResponse<any>): string {
```

Comment:
`<any>`

Reply:
also looks like we need a shared utils package or maybe adding that function to `http-service`

Reply:
This one is `any` on purpose. If different services return errors in the same way then having a shared util would make sense. I ended up putting these in the client specifically so that we could have service specific logic for building error messages from responses. 

Cosmos may return error messages in a different way then the `ui-update-service` etc. so this code could and likely would be slightly different based on which client it is.